### DISCUSSION TITLE
IT Community

### DISCUSSION THREAD

**Discussion by hoangng-003** [community/community#27415] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone , 
I'm trying to find **tech communities** on social media
I tried searching on _Twitter , Github , StackOverFlow , Reddit , ...._ but quite difficult

I want to update the general technology situation in the world easily and share my knowledge with everyone in the IT community

Can anyone help me share some links to famous IT groups and communities on social networks?

**Comment by sawyerbivens** [comment, marked as answer]
I recommend checking LinkedIn, one quick search got me [this](https://www.opensesame.com/site/blog/10-linkedin-groups-it-professionals/) list of groups on Linkedin so I wouldn't be surprised if there were plenty more.

Best of luck finding a group!

**Reply by hoangng-003** [reply]
Oh ty bro .... it seems to be interesting

**Reply by sawyerbivens** [reply]
Yea of course

**Comment by Gisgar3** [comment]
Like @sqwyer recommended, I've found [LinkedIn](https://www.linkedin.com/) to be a great platform to connect with communities in the IT space. You can surprisingly find a lot on there! Hope to see you reach your goals 😃
